### ISSUE TITLE
Exported time is wrong

### ISSUE THREAD

**Issue by Eon9D** [glpi-project/glpi#6035]
**Describe the bug**
```
Exporting "time" as CSV is wrong / unclear.
```
**Page(s) URL**
```
https://glpi.system/front/stat.tracking.php
```
**To reproduce**
```
Steps to reproduce the behavior:
1. Select Tickets -> By ticket on URL
2. Select "All pages as CSV" and download the CSV
3 Open in Excel (etc.) and compare results
```
**Expected behavior**
```
3 minutes should be 180 seconds or 3 minutes ; but is 2.066.510.989.011 (unit?)
8 minuts shoulb be 480 seconds or 8 minutes; but is 53.415.111.153.846 (unit?)
```
**Logs**
```
not relevant
```
**Screenshots**

![image](https://user-images.githubusercontent.com/32479943/59588916-f5abcc00-90e8-11e9-8d98-a3335f6fcea3.png)

**Your GLPI version**

```
9.4.2
```

**Comment by cedric-anne** [comment]
Hi,

First, your issue may be related to formatting options in Excel. I do not have Excel so I cannot reproduce.

On LibreOffice, exporting in a hh:mm:ss format seems to be the best option, as it is human readable and sorting is working properly. Could you try this solution available in #6036 ?

Regards

**Comment by Eon9D** [comment]
Thank you for your quick reply! I am busy right now but I will have a look this evening and let you know.
